src/i18n/pl_PL.ts

```diff
@@ -0,0 +1,1050 @@
+            close: 'Zamknij wiadomość i przeładuj grę (zalecane)',
+            abort: 'Zmaknij wiadomość bez przeładowania',
+        },
+    },
+    globalSettings: {
+        name: 'Ustawienia główne',
+        labelInMenu: {
+            title: 'Etykieta zamiast ikony w menu',
+            description:
+                'Wyświetla prostą etykietę na pasku nawigacji zamiast logo LSSM',
+        },
+    },
+    vehicles: {
+       0: {
+            caption: 'Ciężki samochód gaśniczy',
+            color: '#990000',
+            coins: 25,
+            credits: 5_000,
+            minPersonnel: 2,
+            maxPersonnel: 2,
+            wtank: 5_000,
+        },
+       1: {
+            caption: 'Średni samochód gaśniczy',
+            color: '#990000',
+            coins: 25,
+            credits: 5_000,
+            minPersonnel: 2,
+            maxPersonnel: 2,
+            wtank: 2_500,
+        },
+       2: {
+            caption: 'SD',
+            color: '#990000',
+            coins: 30,
+            credits: 10_000,
+            minPersonnel: 1,
+            maxPersonnel: 1,
+            wtank: 0,
+            special: 'Wymagane po zbudowaniu 3 remiz',
+        },
+       3: {
+            caption: 'SLOp',
+            color: '#990000',
+            coins: 25,
+            credits: 10_000,
+            minPersonnel: 1,
+            maxPersonnel: 1,
+            wtank: 0,
+            special: 'Wymagane po zbudowaniu 6 remiz',
+        },
+       4: {
+            caption: 'Samochód ratownictwa technicznego',
+            color: '#990000',
+            coins: 25,
+            credits: 12_180,
+            minPersonnel: 3,
+            maxPersonnel: 3,
+            wtank: 0,
+            special: 'Wymagane po zbudowaniu 4 remiz',
+        },
+       5: {
+            caption: 'Ambulans P',
+            color: '#FFFFCC',
+            coins: 25,
+            credits: 5_000,
+            minPersonnel: 2,
+            maxPersonnel: 2,
+            wtank: 0,
+        },
+       6: {
+            caption: 'Cysterna gaśnicza',
+            color: '#990000',
+            coins: 25,
+            credits: 17_300,
+            minPersonnel: 1,
+            maxPersonnel: 1,
+            wtank: 14_000,
+            special: 'Wymagane po zbudowaniu 7 remiz',
+        },
+       7: {
+            caption: 'Samochód ratownictwa chemicznego',
+            color: '#990000',
+            coins: 25,
+            credits: 19_200,
+            minPersonnel: 1,
+            maxPersonnel: 1,
+            wtank: 0,
+            schooling: 'Szkoła Pożarnicza',
+            shownSchooling: 'Wymaga specjalnej edukacji (Ratownictwo chemiczne)',

```

Comment:
This need to be the same as in the personal list. 
In the Dutch version is it in this example `Duiker/Duikploegleider`
![afbeelding](https://user-images.githubusercontent.com/65117490/100231235-0475ef80-2f27-11eb-8813-ac29ddd02dc0.png)